charts/ceph-csi-cephfs/values.yaml

```
@@ -331,6 +331,9 @@ secret:
   # specified in the storage class
   adminID: <plaintext ID>
   adminKey: <Ceph auth key corresponding to ID above>
+  # User credentials are required for the static provisioner.
```

```suggestion
  # User credentials are required for the static provisioned PVC.
```
can you please update the same above?

---

Sure, I fixed this occurrence and the two others in the chart README.